src/test-fbp/test-blob-data.txt

```diff
@@ -0,0 +1 @@
+test validator data
```

Comment:
It's one test, of course, but since blobs can accept any kind of binary data, it would be nice to test that as well. That's not going to work with the code as is (`strlen` and `strdup` are used, for instance), but it's something that should be tested.